core/src/test/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotServiceSpec.scala

```diff
@@ -27,15 +26,15 @@ import scala.concurrent.duration.Duration
 
 class SnapshotServiceSpec extends AnyWordSpecLike with Matchers with MockitoSugar with TestDriverComponent {
 
-  implicit val cs = IO.contextShift(global)
+  implicit val cs: ContextShift[IO] = IO.contextShift(global)
 
   //test constants
-  val fakeRawlsClientEmail = WorkbenchEmail("fake-rawls-service-account@serviceaccounts.google.com")
-  val fakeDeltaLayerStreamerEmail = WorkbenchEmail("fake-rawls-service-account@serviceaccounts.google.com")
+  val fakeRawlsClientEmail: WorkbenchEmail = WorkbenchEmail("fake-rawls-service-account@serviceaccounts.google.com")
+  val fakeDeltaLayerStreamerEmail: WorkbenchEmail = WorkbenchEmail("fake-rawls-service-account@serviceaccounts.google.com")
```

Comment:
Why are you specifying the types on the lhs here?  I feel like I was corrected when I did that at one point :)  Just curious.

Reply:
heh - IntelliJ throws (ignorable) warnings if these aren't present, and I was trying to make IntelliJ happy. It does make the Scala more verbose … hey, maybe I should just turn off that warning in IntelliJ …

Reply:
I prefer the verbosity personally (the IDE does all the the typing anyhow), but probably best to stick with whatever preponderant pattern is in the code base.